objectRecognitionSource/src/test/java/io/vantiq/extsrc/objectRecognition/neuralNet/TestYoloProcessor.java

```diff
@@ -125,10 +232,44 @@ public void testResults() throws ImageProcessingException {
     @Test
     public void testRealJSONConfig() throws ImageProcessingException, JsonParseException, JsonMappingException, IOException {
         YoloProcessor ypImageSaver = new YoloProcessor();
-        ExtensionServiceMessage msg = createRealConfig();
+        ExtensionServiceMessage msg = createRealConfig(neuralNetJSON1);
         Map config = (Map) msg.getObject();
         Map neuralNetConfig = (Map) config.get("neuralNet");
         
+        // Config with meta file, label file, pb file, and anchors
+        try {
+            ypImageSaver.setupImageProcessing(neuralNetConfig, SOURCE_NAME, MODEL_DIRECTORY, testAuthToken, testVantiqServer);
+        } catch (Exception e) {
+            fail("Should not fail with valid config.");
+        }
+        
+        // Config with meta file and pb file
+        msg = createRealConfig(neuralNetJSON2);
+        config = (Map) msg.getObject();
+        neuralNetConfig = (Map) config.get("neuralNet");
+        
+        try {
+            ypImageSaver.setupImageProcessing(neuralNetConfig, SOURCE_NAME, MODEL_DIRECTORY, testAuthToken, testVantiqServer);
+        } catch (Exception e) {
+            fail("Should not fail with valid config.");
+        }
+        
+        // Config with label file, pb file, and anchors
+        msg = createRealConfig(neuralNetJSON3);
+        config = (Map) msg.getObject();
+        neuralNetConfig = (Map) config.get("neuralNet");
+        
+        try {
+            ypImageSaver.setupImageProcessing(neuralNetConfig, SOURCE_NAME, MODEL_DIRECTORY, testAuthToken, testVantiqServer);
+        } catch (Exception e) {
+            fail("Should not fail with valid config.");
+        }
+        
+        // Config with label file and pb file
+        msg = createRealConfig(neuralNetJSON4);
+        config = (Map) msg.getObject();
+        neuralNetConfig = (Map) config.get("neuralNet");
+        
```

Comment:
These tests only accept that we correctly set internal variables & classes.  No actual processing is done.